engine/order_manager.go

```diff
@@ -589,6 +589,14 @@ func (m *OrderManager) processSubmittedOrder(newOrderResp *order.SubmitResponse)
 		return nil, err
 	}
 
+	if err := m.orderStore.add(detail.CopyToPointer()); errors.Is(err, ErrOrdersAlreadyExists) {
+		// Streamed by ws before we got here. Details from ws supersede since they are more recent.
+		detail = m.orderStore.getByDetail(detail)
+	} else if err != nil {
+		// Non-fatal error: Unable to store order, but error does not need to be returned to caller
+		log.Errorf(log.OrderMgr, "unable to add %v order %v to orderStore: %s", detail.Exchange, detail.OrderID, err)
```

Comment:
```suggestion
		log.Errorf(log.OrderMgr, "Unable to add %v order %v to orderStore: %s", detail.Exchange, detail.OrderID, err)
```

Pedantic nit, to be consistent with all other log messages now that this isn't returned as an error

Reply:
Thank you. Fixed [c8235a5](https://github.com/thrasher-corp/gocryptotrader/pull/1336/commits/c8235a56b7223f0d3020daf1e7f84c2cdd95a663)